src/pybind/mgr/dashboard/frontend/src/app/shared/components/telemetry-notification/telemetry-notification.component.html

```diff
@@ -2,8 +2,11 @@
            type="warning"
            *ngIf="displayNotification"
            (close)="close($event)">
-  <div i18n>The Telemetry module is not submitting telemetry data at the
-    moment. Click
-  <a routerLink="/telemetry"
-     class="alert-link">here</a> to activate it now.</div>
+  <div i18n>The Ceph community needs your help to continue improving: please
+    <button class="activate-button">
```

Comment:
Just a suggestion: we can get rid of button element and simply apply the classes in the link,
something like this (example):
```
--- a/src/pybind/mgr/dashboard/frontend/src/app/shared/components/telemetry-notification/telemetry-notification.component.html
+++ b/src/pybind/mgr/dashboard/frontend/src/app/shared/components/telemetry-notification/telemetry-notification.component.html
@@ -3,10 +3,9 @@
            *ngIf="displayNotification"
            (close)="close($event)">
   <div i18n>The Ceph community needs your help to continue improving: please
-    <button class="activate-button">
     <a  routerLink="/telemetry"
-        class="alert-link activate-text">Activate</a></button> the
-    <a href="https://docs.ceph.com/en/latest/mgr/telemetry/">Telemetry </a>module
+        class="btn activate-button alert-link activate-text">Activate</a> the
+    <a href="https://docs.ceph.com/en/latest/mgr/telemetry/">Telemetry</a> module
   </div>
 </ngb-alert>
 
diff --git a/src/pybind/mgr/dashboard/frontend/src/app/shared/components/telemetry-notification/telemetry-notification.component.scss b/src/pybind/mgr/dashboard/frontend/src/app/shared/components/telemetry-notification/telemetry-notification.component.scss
index 3a6823ca1b2..569f1e0c4a4 100644
--- a/src/pybind/mgr/dashboard/frontend/src/app/shared/components/telemetry-notification/telemetry-notification.component.scss
+++ b/src/pybind/mgr/dashboard/frontend/src/app/shared/components/telemetry-notification/telemetry-notification.component.scss
@@ -8,6 +8,7 @@
   background-color: vv.$barley-white;
   border: vv.$gray-700 solid 0.5px;
   border-radius: 10%;
+  padding: 0.1rem 0.4rem;
 }

```

Reply:
Done